packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -1,13 +1,34 @@
 :local {
   .searchfield {
-    display: inline-flex;
+    display: flex;
+    min-width: 0;
+    width: var(--terra-search-field-max-width, 15.31em);
+
+    .clear {
+      background: transparent;
+      border-bottom-left-radius: var(--terra-search-field-clear-border-bottom-left-radius, 0);
+      border-bottom-right-radius: var(--terra-search-field-clear-border-bottom-right-radius, 0);
+      border-color: var(--terra-search-field-clear-border-color, #dedfe0);
+      border-left: var(--terra-search-field-clear-border-left, 0);
+      border-right: var(--terra-search-field-clear-border-right, 0);
```

Comment:
Let's set this default value to `1px solid rgba(0,0,0,0)`. This should help prevent a pixel shift when we add the focus style.

Reply:
I don't think that will work. Borders come in at a diagonal, so you will this gap with the style. 
<img width="378" alt="Screen Shot 2019-04-08 at 4 28 44 PM" src="https://user-images.githubusercontent.com/2552573/55758125-7a100d80-5a1b-11e9-8c7b-c9339a431d4f.png">

Most likely, it needs to be the same color as the rest of the input border

Reply:
This should be resolved here: [83eee68](https://github.com/cerner/terra-core/pull/2317/commits/83eee6882b20fa5c793d89e7eea8d429e46ebd52)